packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -59,97 +191,101 @@ export default class TradeStore extends BaseStore {
     has_equals_only = false;
 
     // Underlying
-    symbol;
+    symbol = '';
     is_market_closed = false;
     previous_symbol = '';
-    active_symbols = [];
+    active_symbols: ActiveSymbols = [];
 
-    form_components = [];
+    form_components: string[] = [];
 
     // Contract Type
     contract_expiry_type = '';
     contract_start_type = '';
     contract_type = '';
-    contract_types_list = {};
-    trade_types = {};
+    contract_types_list: TContractTypesList = {};
+    trade_types: { [key: string]: string } = {};
 
     // Amount
     amount = 10;
     basis = '';
-    basis_list = [];
+    basis_list: Array<TTextValueStrings> = [];
     currency = '';
-    stake_boundary = {};
+    stake_boundary: TStakeBoundary = { VANILLALONGCALL: {}, VANILLALONGPUT: {} };
```

Comment:
@henry-deriv please keep master version of stake_boundary as `stake_boundary: Partial<TStakeBoundary> = {};`. It became relevant after Turbos PR https://github.com/binary-com/deriv-app/pull/7514/files#diff-ac84bdaae0a85981966f330c025656c3aaa66895a9376e738bb503d088594ec0 was released.

Reply:
done